Task_One/Firest_task.rb

```diff
@@ -0,0 +1,20 @@
+OPTIMAL = 110
```

Comment:
the file name should be corresponding with the class name

Reply:
I changed file name from Firest_task to IdealWeight